src/main/java/com/puppycrawl/tools/checkstyle/checks/design/ThrowsCountCheck.java

```diff
@@ -70,13 +104,72 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public void failTest() throws IllegalStateException,
+ *                                 ArrayIndexOutOfBoundsException,
+ *                                 NullPointerException { // violation, max allowed is 2
+ *         // body
+ *     }
+ *
+ *     public void successTest() throws ArithmeticException,
+ *                                 NumberFormatException { // ok
+ *         // body
+ *     }
+ *
+ *     private void privateTest() throws CloneNotSupportedException,
+ *                                 ClassNotFoundException,
+ *                                 IllegalAccessException,
+ *                                 ArithmeticException,
+ *                                 ClassCastException { // ok, private methods are ignored
+ *         // body
+ *     }
+ *
+ * }
+ * </pre>
+ * <p>
  * To configure the check so that it doesn't skip private methods:
  * </p>
  * <pre>
  * &lt;module name="ThrowsCount"&gt;
  *   &lt;property name=&quot;ignorePrivateMethods&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public void failTest() throws CloneNotSupportedException,
+ *                                 ArrayIndexOutOfBoundsException,
+ *                                 StringIndexOutOfBoundsException,
+ *                                 IllegalStateException,
+ *                                 NullPointerException { // violation, max allowed is 4
+ *         // body
+ *     }
+ *
+ *     public void successTest() throws ArithmeticException,
+ *                                 NumberFormatException { // ok
+ *         // body
+ *     }
+ *
+ *     private void privateTest() throws CloneNotSupportedException,
+ *                                 ClassNotFoundException,
+ *                                 IllegalAccessException,
+ *                                 ArithmeticException,
+ *                                 ClassCastException { // violation, max allowed is 4
+ *         // body
+ *     }
+ *
+ *     private void successTest() throws IllegalStateException,
```

Comment:
there are two `successTest()` methods in this example.

maybe these examples could be improved with more realistic names? I don't mean to spend a lot of time on it, but instead of `Test`, `failTest`, `successTest` etc.

Reply:
Hey @esilkensen changed the function names

Reply:
I was wondering about something a little more like you might see in real code or Java tutorials or that sort of thing, like instead of `Test`, `myFunction`, and `myFunc` maybe `Writer`, `writeList`, etc. -- again, not meaning to spend a lot of time on it, and I will also pass on to the next reviewer for their thoughts, if they are OK as is, then I am too.

Reply:
There is no functionality i am performing in those functions so i gave that name @esilkensen

Reply:
right, I don't mean to suggest to include functionality or implementation of the methods, only to think for illustrative purposes of the examples if there are names that may be better than `myFunction` and `myFunc`. after CI passes I will pass on to another reviewer.